### PR TITLE
SAK-42096 - Mark as read message icon not displaying in forums

### PR DESCRIPTION
**Pull request by adrianmticarum**
Commit to show the "Mark as read" icon and update unread messages count
in the conversation.

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/js/forum.js [hunk 1]
```diff
@@ -374,6 +374,10 @@ function doAjax(messageId, topicId, self){
                     if ($(self).parent('td').size() === 1) {
                         var thisTheadClassArr = $(thisRow).prop('class').split(' ');
                         var thisThread = thisTheadClassArr[thisTheadClassArr.length - 1];
+                        var unread = parseInt($('.hierItemBlock.' + thisThread + ' .childrenNewNumber').text());
+                        if(unread > 0) {
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forum.js]: Small thing. Can you add a space after the if? It makes the code a bit clearer.

Comment (adrianmticarum) [reply msgcntr/messageforums-app/src/webapp/js/forum.js]: Done!

**Comment by adrianmticarum** [comment]
> Do you need to do the same thing with dfForumDetail and dfAreaInclude?

No, we don´t need it as there is no "mark as read" button in either of these pages, but I have realized that the previous change was not the best solution for this bug.

In "dfViewThreadBodyInclude.jsp" the "mark as read" button was using the "trans.gif" icon too, but inside a commandLink with the class "markAsReadIcon". In this jsp (in 12.X) the button works perfecty. When I tried to copy this button to "dfAllMessages.jsp" in master, it should have worked too, but there was a problem with the class in master. As a result I have had to change the class too in order to make opperative the button in both pages, so this is a 2x1!

**Comment by adrianmticarum** [comment]
Hi @adrianfish ! The changes are ready now, we have had to deal with some Codacy issues. Please, take a look at this PR when you have a moment. Thanks!

**Comment by adrianmticarum** [comment]
Yes. there's a test plan on the SAK.

**Comment by adrianmticarum** [comment]
Hi again @adrianfish ! We have comitted a new change in the JS to fix a problem with the parseInt function.

**Comment by adrianfish** [comment]
@adrianmticarum Good spot on the parseInt thing. I should have caught that !

**Pull request merged**